ngraph/frontend/onnx_import/src/onnx.cpp

```diff
@@ -96,6 +96,34 @@ namespace ngraph
                 }
             }
 
+            // The paths to external data files are stored as relative to model path.
+            // The helper function below combines them and replaces the original relative path.
+            // As a result in futher processing data from external files can be read directly.
+            void update_external_data_paths(ONNX_NAMESPACE::ModelProto& model_proto,
+                                            const std::string& model_path)
+            {
+                const auto model_dir_path = model_path.substr(0, model_path.find_last_of(R"(/)"));
```

Comment:
I think this can be problematic for non-unix systems. Let's see what happens in CI.

Reply:
Good point, it seems to work correctly in CI, but I used now features from `ngraph/file_util.hpp`